# HG changeset patch
# User msirringhaus@suse.de
# Date 1558452418 -7200
#      Tue May 21 17:26:58 2019 +0200
# Node ID a56e9eb77d0ecd64d5aa180a71a7c76c35b50961
# Parent  65dee647ea92d0c21366e5673c0d833376561c7e
imported patch mozilla-ppc-altivec_static_inline.patch

diff --git a/gfx/qcms/transform-altivec.c b/gfx/qcms/transform-altivec.c
--- a/gfx/qcms/transform-altivec.c
+++ b/gfx/qcms/transform-altivec.c
@@ -30,7 +30,7 @@
 static const ALIGN float floatScaleX4 = FLOATSCALE;
 static const ALIGN float clampMaxValueX4 = CLAMPMAXVAL;
 
-inline vector float load_aligned_float(float *dataPtr)
+static inline vector float load_aligned_float(float *dataPtr)
 {
 	vector float data = vec_lde(0, dataPtr);
 	vector unsigned char moveToStart = vec_lvsl(0, dataPtr);
